pact/Pact/Core/IR/Eval/Runtime/Utils.hs

```diff
@@ -71,6 +78,42 @@ import Pact.Core.Guards
 import Pact.Core.Capabilities
 import Pact.Core.Hash
 
+emitReservedEvent :: Text -> [PactValue] -> ModuleHash -> EvalM e b i ()
+emitReservedEvent name params mhash = do
+  let pactModule = ModuleName "pact" Nothing
+  let pe = PactEvent name params pactModule mhash
+  emitEventUnsafe pe
+
+emitEvent
+  :: i
+  -> PactEvent PactValue
+  -> EvalM e b i ()
+emitEvent info pe = findCallingModule >>= \case
+    Just mn -> do
+      -- Todo: ++ definitely feels suboptimal, especially for gas.
```

Comment:
consider moving the comment to `emitEventUnsafe`, as `emitEvent` does not use `++`.